src/main/java/com/ericsson/gerrit/plugins/eiffel/configuration/EiffelPluginConfiguration.java

```diff
@@ -67,12 +69,23 @@ public EiffelPluginConfiguration(final String pluginName, final NameKey project,
         }
         // Read plugin configuration
         this.enabled = pluginConfig.getBoolean(ENABLED, false);
-        this.filter = pluginConfig.getString(FILTER);
+        if (pluginConfig.getStringList(FILTER)!=null && pluginConfig.getStringList(FILTER).length > 0) {
+            this.filter = Arrays.stream(pluginConfig.getStringList(FILTER)).collect(Collectors.joining(","));
+            pluginConfig.setString(FILTER, this.filter);
```

Comment:
What is the point of setting a value you have just read?

Reply:
This is so as to see the other values set manually using config properties.
Once you have edited it manually it will not be visible in the configurations GUI. I have set this so that they are visible in the GUI.